src/main/java/io/dapr/components/aspects/AdvertisesFeatures.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2022 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.dapr.components.aspects;
+
+import java.util.List;
+
+public interface AdvertisesFeatures {
+  /* TODO should we move getFeatures and ping() and emptyNull to their respective
+   *      interfaces so that we can also document what those components characteristics?
+   */
+  List<String> getFeatures();
```

Comment:
is it worth to have TODO's tracked outside code? Like gh issues?

Reply:
Ironically, that TODO is DONE :)